src/sentry_os.c

```diff
@@ -117,7 +117,7 @@ sentry__get_os_context(void)
 #    if defined(_GAMING_XBOX_SCARLETT)
 #        pragma warning(push)
 #        pragma warning(disable : 4996)
-    sentry_value_set_by_key(os, "name", sentry_value_new_string("Windows"));
+    sentry_value_set_by_key(os, "name", sentry_value_new_string("Xbox"));
```

Comment:
Will the version number always match?

This also doesn't tell us which model (One, Series X or S etc)?